src/goto-programs/system_library_symbols.cpp

```diff
@@ -318,5 +339,18 @@ bool system_library_symbolst::is_symbol_internal_symbol(
     return true;
   }
 
+  if(use_all_headers &&
```

Comment:
Agreed re: `$link`, disagreed re: `_assert` et al, which is your addition (https://github.com/diffblue/cbmc/commit/fc756f336c04a788487c2f7e59bd1c8568c4c5f5)